### DISCUSSION TITLE
What is the Correct Way of Getting the Token Both Android and IOS

### DISCUSSION THREAD

**Discussion by janpaulalmanoche** [invertase/react-native-firebase#4642] [Q&A] [answered]
is this correct? im getting token using this line of code

` PushNotification.configure({
 onRegister: function (token) {
        console.log(' APP JS TOKEN:', token);
        }
})`

im using this package
https://github.com/zo0r/react-native-push-notification

**this is an example of the token generated in my ios actual device,is this one valid? it is shorter than the one that was generated in android**
![image](https://user-images.githubusercontent.com/44863381/100980973-112ac100-3581-11eb-98da-6fe8db247690.png)

**this is the token generated in android**
![image](https://user-images.githubusercontent.com/44863381/100981072-3dded880-3581-11eb-851f-80b2b79041ad.png)

**Comment by janpaulalmanoche** [comment]
im using this and it works fine in Android, im able to receive notification in all App States, but i wonder if this will work on IOS

**Reply by mikehardy** [reply]
In general: test it and see

**Comment by janpaulalmanoche** [comment, marked as answer]
to clarify things out it works for both Android and IOS
